sdk/python/packages/flet/src/flet/core/video.py

```diff
@@ -96,6 +96,7 @@ def __init__(
         on_error: OptionalControlEventCallable = None,
         on_completed: OptionalControlEventCallable = None,
         on_track_changed: OptionalControlEventCallable = None,
+        on_percent_changed: OptionalControlEventCallable = None,
```

Comment:
"percent" seems quite ambiguous here

Reply:
I originally had it as on_pos_changed, but that seemed ambiguous to what it was doing, so I named it on_percent_changed to not confuse it with giving durations but giving percentages for the flet slider.

Reply:
Actually, why send position values as percentage instead? Why not send "number of seconds" directly for example? Those who want percentage-conversion could then do it freely on python end. 

We could then name this event `on_position_change`, as the goal here is to share details on the new position.
What do you think?

(will appreciate test code to better review your PR)

Reply:
When I looked at my slider config for displaying position changes in flet code went like:
```
        self.slider1 = ft.Slider(
            value=0,
            min=0,
            max=100,
            width=100,
            tooltip="Seek",
            divisions=100,
            label="{value}%",
            on_change=self.seek,
            thumb_color=ft.Colors.PRIMARY_CONTAINER,
            disabled=False,
            active_color=ft.Colors.ON_PRIMARY_CONTAINER,
            inactive_color=ft.Colors.ON_PRIMARY,
            on_focus=self.slider_focus,
            on_blur=self.slider_blur,
            on_change_end=self.on_change_end,
        )
```

I have 100 positions it can update with from 0-100 which seemed more than adequate, the issue is ft.Slider wants it in this format so then I was forced to do something like this in python:

```
                    duration = await self.config.video.get_duration_async(
                        wait_timeout=5
                    )
                    pos = await self.config.video.get_current_position_async(
                        wait_timeout=5
                    )
                    percent = int(pos / duration * 100)
                    print(f"Position changed to {pos} and duration is: {duration} Percent is: {percent}")
                    await self.config.video_lock.acquire()  # so we don't update slider while they're selecting a seek
                    self.slider1.value = percent
```

This seemed incredibly inefficient as I was using python to do the conversions, when we could use a faster language like dart to do the conversion for us then just send it to us.

Reply:
I could create another handler for durations if that is what people want, but I think this is most efficient of working with ft.Slider.

Reply:
I see what your saying though, looking at a music app from playstore, it keeps displaying time its been playing for the track.

Reply:
What if passed duration, position and percentage, then everyone wins and they can do what they want with the values since they will always have all those in their flet program stored as variables just by enabling the handler? Also make the throttlespeed a configurable in milliseconds somewhere else.

Reply:
> duration, position, percent

Sure, we can do it that way. Will you work on it? (Inspiration: [py](https://github.com/flet-dev/flet/blob/b842dc338ce128d5e328f98de615aa20d3b7c561/sdk/python/packages/flet/src/flet/core/dismissible.py#L299-L306), [dart](https://github.com/flet-dev/flet/blob/ab58bf9944fd6f2f5e69aae2417d31f2125b8f3a/packages/flet/lib/src/controls/interactive_viewer.dart#L134-L143))